Assets/MixedRealityToolkit/Utilities/CameraCache.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System.Runtime.InteropServices.WindowsRuntime;
```

Comment:
What is this used for?

Reply:
it isnt. got added accidentally. nice catch!